newIDE/electron-app/app/main.js

```diff
@@ -52,6 +52,7 @@ protocol.registerSchemesAsPrivileged([{ scheme: 'gdide' }]);
 // working in the renderer process.
 // See https://github.com/electron/electron/issues/22119
 // For now, disable this as we rely heavily on `fs` in the renderer process.
+// This is also done in the runtime: GDJS/Runtime/Electron/main.js
```

Comment:
I think you can remove it, if someone update the runtime or the IDE to a new electron version, it must be done in two separate package.json, so it make sense for this person to go through the two main.js file to verify if there is anything to upgrade :)